modules/ROOT/pages/amazon/amazon-sqs-connector-examples.adoc

```diff
@@ -0,0 +1,297 @@
+			"dataType" : "Number"
+		} as Object {
+			class : "org.mule.extension.sqs.api.model.MessageAttributeValue"
+		}
+	} as Object {
+		class: "java.util.HashMap"
+	}
+} as Object {
+	class: "org.mule.extension.sqs.api.model.Message"
+}
+----
++
+image::amazon/amazon-sqs-transform-message.png[transform-message]
+
+=== Add and Configure the SQS Send Message Operation
+
+. In the Mule Palette, search for *SQS* and select the *Send message* operation.
++
+image::amazon/amazon-sqs-select-send.png[select-send-message]
++
+. Drag the *Send message* operation onto the canvas, to the right of the *Transform Message* component.
+. In the *Send message* configuration, click *+* next to the *Connector configuration* field to add a global element.
+. Configure the global element as follows:
++
+[%header%autowidth.spread]
+|===
+|Field |Value
+|Name |Name used to reference the configuration.
+|Access Key|Alphanumeric text string that uniquely identifies the user who owns the account.
+|Secret Key |Key that plays the role of a password.
+|Region endpoint | Queue region
+|===
++
+For example:
++
+image::amazon/amazon-sqs-send-global-config.png[send-global-config]
++
+. Select the *Configuration XML* tab to view the corresponding XML:
++
+[source,dataweave,linenums]
+----
+<sqs:config name="Amazon_SQS_Configuration"
+  doc:name="Amazon SQS Configuration"
+	defaultQueueUrl="${sqs.queueUrl}">
+	<sqs:basic-connection accessKey="${sqs.accessKey}"
+	secretKey="${sqs.secretKey}"
+	region="us-east-1"/>
+</sqs:config>
+----
++
+. Configure the Following fields on the properties window:
++
+[%header%autowidth.spread]
+|===
+|Field |Value
+|Display Name |Enter a name for the connector operation.
+|Connector Configuration |Select the global configuration you just created.
+|Message |`#[payload]`
+|Queue url |Enter the queue URL, if you did not enter a default queue URL in the global element.
+|===
++
+For example:
++
+image::amazon/amazon-sqs-send-message.png[send-message]
+
+=== Add a Logger Component to Display the Response in the Mule Console
+
+. In the Mule Palette, search for *Logger*.
+. Drag the component onto the canvas, next to the *Send Message* component.
+. Configure the following fields:
++
+[%header%autowidth.spread]
+|===
+|Field |Value
+|Display Name |Enter a name for the logger, such as `Log Response`.
+|Message |Sent Message: `#[payload]`
+|Level |INFO (Default)
+|===
++
+For example:
++
+image::amazon/amazon-sqs-logger.png[logger]
+
+=== Obtain the Number of Messages in the Queue
+
+. In the Mule Palette, search for *Amazon SQS*.
+. Select the *Get approximate number of messages* operation and drag it onto the canvas, next to the *Logger* component.
++
+. Configure the following fields on the properties window:
++
+[%header%autowidth.spread]
+|===
+|Field |Value
+|Display Name |Enter a name for the connector operation.
+|Connector Configuration |Select the global configuration you created previously for Amazon SQS Connector.
```

Comment:
This configuration is not the configuration which **Get Approximate number of messages** operation has. Only  field **queue url** should be there